lib/tarantool_server.py

```diff
@@ -144,6 +147,12 @@ def execute(self, server):
             self.run_params
         )
         self.inspector.set_parser(ts)
+
+        if self.suite_ini.get('pre_cleanup', False):
```

Comment:
So it will work only for 'core = tarantool' tests. What about 'core = app'?

Reply:
Yes it will works only for 'core = tarantool' test. As I mention above 'core = app' not needed cleanup.